config/routes.rb

```diff
@@ -88,7 +88,7 @@ def matches?(request)
     get "/auth/failure", to: "auth#failure"
 
     resources :claims, only: [:index, :show] do
-      resources :checks, only: [:index, :show, :create], param: :check, constraints: {check: %r{#{Admin::ChecksController::CHECKS_SEQUENCE.join("|")}}}
+      resources :tasks, only: [:index, :show, :create], param: :task, constraints: {task: %r{#{Admin::TasksController::TASKS_SEQUENCE.join("|")}}}
```

Comment:
```suggestion
      resources :tasks, only: [:index, :show, :create], param: :name, constraints: {name: %r{#{Admin::TasksController::TASKS_SEQUENCE.join("|")}}}
```

I'm wondering if it's better to call the `task` param `name`, as this better reflects the mapping in the database? It also means, when we get to the yes/no work, we can have a `task` param hash, which is more in line with Rails' defaults.

Reply:
Makes sense, I'll update it.

Reply:
Renamed, as requested: https://github.com/DFE-Digital/dfe-teachers-payment-service/compare/98e21787586d404cd621af73f55ca394039a2354..fe9802e2d27814c178aacfa63f5c241463fe235b